src/main/java/org/ngafid/ProcessUpload.java

```diff
@@ -433,4 +489,39 @@ public static boolean ingestFlights(Connection connection, int uploadId, int fle
         //ingestion was successfull
         return true;
     }
+
+    private static void placeInZip(String file, String zipFileName) throws IOException {
+        System.out.println("Placing " + file + " in zip");
```

Comment:
I would advise using the `LOG` here for sysouts. Makes the output more uniform and traceable.

Reply:
I will do this, but the entire ProcessUpload file uses Sysouts instead of a Logger which was the reasoning behind this